### ISSUE TITLE
basher link, improve error message on bad format

### ISSUE THREAD

**Issue by nhomble** [basherpm/basher#102]
I am running the `basher link <dir> <package>` command

I incorrectly defined the package argument since I didn't have the namespace argument. The error message just led me to the default usage function
```
Usage: basher link [--no-deps] <directory> <package>

Installs a local directory as a basher package
```

It wasn't until I added my own debug statements to the `basher-link` file that I figured out with `-z` check was failing. 

I think it would be helpful if the error messages told me which variable is incorrectly empty.

**Comment by juanibiapina** [comment]
PRs are welcome :)

**Comment by nhomble** [comment]
If you like the feature, I don't mind adding some helpful logging.

Should the message go to stderr?

**Comment by juanibiapina** [comment]
Sorry I don't remember anymore if these types of messages are going to stderr or stdout, but you can probably find a similar case in code.

**Comment by juanibiapina** [comment]
Closed by #103
